bamboo/create-release-artifacts.sh

```diff
@@ -4,7 +4,8 @@ set -ex
 
 ## Build TF modules that require source building
 
-(cd tf-modules/distribution && ./bin/build-tf-module.sh && cp ./dist/terraform-aws-cumulus-distribution.zip ../../terraform-aws-cumulus-distribution.zip)
+(cd tf-modules/distribution && ./bin/build-tf-module.sh && cp ./dist/terraform-aws-tea-distribution.zip ../../terraform-aws-tea-distribution.zip)
+(cd tf-modules/cumulus_distribution && ./bin/build-tf-module.sh && cp ./dist/terraform-aws-cumulus-distribution.zip ../../terraform-aws-cumulus-distribution.zip)
```

Comment:
Talked about this with Katie. There might be an issue here if you're coming from a previous release and you already have a `terraform-aws-cumulus-distribution.zip`. In that case, because we're repurposing that name for the new distribution API (not TEA) it could get confusing or actually cause the wrong distribution API to be included.

Reply:
I added `rm -f ./dist/terraform-aws-cumulus-distribution.zip` here. This would only really remove a file once, after you've moved from a prior release to this one. After that the `-f` will suppress the "file not found" error that `rm` would generate.

I don't _love_ this but I also don't see a huge drawback. It's really just about clean up. There isn't a chance that the old .zip would be release because we're copying the new one to the root directory for release, not the old one.

Reply:
I'm confused - users aren't running this.

Reply:
Explained in a sidebar, this is for local development testing situations where jumping back/forth on releases leaves a useless .zip in the dev env.  This shouldn't happen in CI.